src/System.Net.Requests/src/System/Net/HttpWebRequest.cs

```diff
@@ -384,24 +384,7 @@ public string TransferEncoding
         }
 
 
-        public bool KeepAlive
-        {
-            get
-            {
-                return _webHeaderCollection[HttpKnownHeaderNames.KeepAlive] == bool.TrueString;
-            }
-            set
-            {
-                if (value)
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.TrueString);
-                }
-                else
-                {
-                    SetSpecialHeaders(HttpKnownHeaderNames.KeepAlive, bool.FalseString);
-                }
-            }
-        } 
+        public bool KeepAlive { get; set; } = true;
```

Comment:
Ah.  It means that it's an auto-prop that's initialized to true.  The compiler generates something like:
```C#
private bool _backingFieldKeepAlive = true;

public bool KeepAlive
{
    get { return _backingFieldKeepAlive; }
    set { _backingFieldKeepAlive = value; }
}
```